README.md

```diff
@@ -10,6 +10,8 @@
 
 Our templates offer a rich boilerplate to jump start Go Gin-based application development.
 
+Check this [wiki](https://github.com/nimblehq/gin-templates/wiki/Directories) for more information about template structure.
```

Comment:
Minor syntax issue:

```suggestion
Check this [wiki](https://github.com/nimblehq/gin-templates/wiki/Directories) for more information about the template structure.
```

Reply:
Fixed in 9fd0761bd4fafc96779ed8c62e32903e84f2c583